### PR TITLE
119 prem section card

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

**Comment by avirankatz** [comment]
@yishain11 Following our conversation, waiting for your cue before reviewing this

File: frontend/src/pages/AgreementPage.tsx [hunk 1]
```diff
@@ -79,10 +124,36 @@ const AgreementPage: FC = () => {
...
+                  # {chapterName}
+                </Typography>
+                <Typography fontSize="0.87rem" fontWeight="noraml" color="text.secondary">
+                  {Object.keys(chapter).length} sections
+                </Typography>
+                <Typography fontSize="0.87rem" fontWeight="noraml" color="text.secondary">
+                  {calcChapterSuggestions(chapter)} suggestions
+                </Typography>
+              </Stack>
+              <Stack direction="column" rowGap="2rem" width="70%">
```
Comment (avirankatz) [inline frontend/src/pages/AgreementPage.tsx:147]: No need for width here. Section cards should take available space.
Instead - The Stack containing all of the chapters should use `maxWidth="md"`

**Review by avirankatz — CHANGES_REQUESTED** [review]
General notes:
- [ ] When using `<Typography>` - use variants instead of sx as much as possible
- [ ] Avoid hard-coded strings, use `t(...)`
- [ ] gaps, spacing, margin, padding should use multiples of 4px
- [ ] arrows should be reversed in RTL mode
- [ ] No need for SectionCard directory

**Comment by avirankatz** [comment]
SectionCard.tsx should be directly under components directory

**Review by avirankatz — CHANGES_REQUESTED** [review]
Much better. See minor requested changes

File: frontend/src/pages/AgreementPage.tsx [hunk 2]
```diff
@@ -4,31 +4,78 @@ import { FC, useContext } from 'react';
...
         direction="row"
         sx={{
-          borderBottom: ' 1px solid #3f4550',
+          borderBottom: '1px solid #3f4550',
           paddingBottom: '1.5rem',
         }}
       >
         <Breadcrumbs separator="›" aria-label="breadcrumb">
           <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
             <Typography
```
[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:76]: done

File: frontend/src/pages/AgreementPage.tsx [hunk 3]
```diff
@@ -39,9 +86,9 @@ const AgreementPage: FC = () => {
           <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
             <Stack
               direction="row"
```
[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:88]: used spacing

File: frontend/src/components/SectionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  };
+
+  const backwardsSuggestion = () => {
+    if (suggestionIndex - 1 >= 0) {
+      const newIndex = suggestionIndex - 1;
+      updateContent(newIndex);
+    }
+  };
+
+  const CheckedIconRender = (props: { isSelected: boolean }) => {
```
[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/components/SectionCard.tsx:38]: done

File: frontend/src/components/SectionCard.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            >
+              {t(StringBank.SECTION_CARD_CONTENT_SECTION_NAME, { sectionNum: props.id + 1 })}
+            </Typography>
+            <Typography variant="body2" color="text.secondary">
+              {t(StringBank.SECTION_CARD_CONTENT_VERSIONS, {
+                versionNum: suggestionIndex + 1,
+                totalVersionsNum: props.suggestions.length,
+              })}
+            </Typography>
+            <CheckedIconRender isSelected={true} />
```
[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/components/SectionCard.tsx:95]: done

**Review by avirankatz — APPROVED** [review]

**Pull request merged**